nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_width.py

```diff
@@ -335,7 +335,7 @@ def _generate_width_list(max_width: int, params: ElasticWidthParams) -> List[int
                 if p.max_num_widths == len(width_list):
                     break
                 if 0 >= multiplier > 1:
-                    nncf_logger.warning("Wrong value for multiplier: {}. Skipping ".format(multiplier))
+                    nncf_logger.warning(f"Wrong value for multiplier: {multiplier}. Skipping ")
```

Comment:
I'd throw an exception, as it happens on init. 
@jpablomch

Reply:
as well as in 261 line:
```python
 nncf_logger.warning(f"Width list for {node_name} "
                                    f"contains invalid values: {fixed_width_list}, {max_width}")
```

and line 265:
```python
 nncf_logger.warning(f"Max width for {node_name} is not aligned with pre-trained model")
```

Reply:
@ljaljushkin Thanks! It is ok to throw exceptions.

Reply:
Will fix in a follow-up PR

Reply:
#1431